frontend/src/store/objectStore.ts

```diff
@@ -81,7 +81,7 @@ export const useObjectStore = defineStore('object', () => {
 
     try {
       appStore.beginIndeterminateLoading();
-      await Promise.all(
+      return await Promise.all(
```

Comment:
What's the return value when we catch/throw? The return here may also interfere with the finally block from executing. As this is a function signature change (this function used to return void, now it's returning a Promise) we need to make sure this is actually working on all pathways.